tests/data_context/store/test_ge_store_backends.py

```diff
@@ -0,0 +1,243 @@
+from collections import OrderedDict
+from unittest.mock import patch
+
+import pytest
+
+from great_expectations.data_context.store import GeCloudStoreBackend
+from great_expectations.data_context.store.ge_cloud_store_backend import (
+    GeCloudRESTResource,
+)
+from great_expectations.data_context.types.base import CheckpointConfig
+
+
+def test_GeCloudStoreBackend(
```

Comment:
As this file is only testing the `GeCloudStoreBackend`, can we rename the file to be something like `test_ge_cloud_store_backend`? I want to make sure it's very clear that this is a Cloud-specific file. Additionally, can we mark each test with `@cloud`? Finally, we could probably split up this test here into multiple unit tests (but that's out of scope here).

Reply:
Looked like just a move but agree with @cdkini 's suggestions ☝️

Reply:
Yeah, this is just a move, nothing has changed here. I had originally thought I was going to put the new test here as well but I didn't end up doing that.
It might be cleaner if I dropped this commit from the PR and do as part of the follow-up.